app/controllers/iiif_controller.rb

```diff
@@ -71,7 +71,7 @@ def set_image_response_headers
   end
 
   def set_attachment_content_disposition_header
-    response.headers['Content-Disposition'] = "attachment;filename=#{params[:identifier]}.#{params[:format]}"
+    response.headers['Content-Disposition'] = "attachment;filename=#{identifier_params[:file_name]}.#{params[:format]}"
```

Comment:
Wait, we're assuming the druid is already present in the file name? That's certainly not the case with all of our image objects (see https://purl.stanford.edu/pt257cr7719, but there's even worse cases like "00001.jp2" floating around too )